spec/sales_engine_spec.rb

```diff
@@ -1,22 +1,57 @@
 require 'CSV'
 require 'rspec'
 
+require './data/invoice_mocks'
 require './data/item_mocks'
 require './data/merchant_mocks'
+require './data/Mockable'
 require './lib/file_io'
+require './lib/invoice_repository'
 require './lib/item_repository'
 require './lib/merchant_repository'
 require './lib/sales_engine'
 require './lib/sales_analyst'
+require './lib/transaction_repository'
+require './lib/invoice_item_repository'
```

Comment:
Here too -- @NoahZinter do we need to require customer and customer repo files here?